Sem. 10/Pract.11/Task14.cpp

```diff
@@ -0,0 +1,40 @@
+#include <iostream>;
+
+char toLower(char ch)
+{
+    return (ch >= 'A' && ch <= 'Z') ? ch + ('a' - 'A') : ch;
+}
+
+
+bool isPrefix(const char* pattern, char* text) {

```

Comment:
Not working in every situation! Take for account this input:

```c++
"Hoqdy! How are you? How was your day?"
"How"
```

Here the output should be:

```c++
"Hoqdy! *** are you? *** was your day?"
```

But instead the output is:

```c++
**qdy! *** are you? *** was your day?
```